code/core/src/common/js-package-manager/Yarn1Proxy.ts

```diff
@@ -36,7 +36,7 @@ export class Yarn1Proxy extends JsPackageManager {
 
   getInstallArgs(): string[] {
     if (!this.installArgs) {
-      this.installArgs = ['--ignore-workspace-root-check'];
```

Comment:
This one is puzzling... but this flag seemed to not be allowed sometimes, even though it's yarn 1.

I do not have an explanation for why... but I do know the flag is never useful for sandbox we generate, because those are never generated in a monorepo, but always in a temp directory, in isolation.